tools/amo/build.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+#
+# Build script for AMO reviewers
+#
+# Ghostery Browser Extension
+# http://www.ghostery.com/
+#
+# Copyright 2019 Ghostery, Inc. All rights reserved.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0
+
+set -e
+
+# Set directory paths for both projects
+GHOSTERY_SOURCE_ZIP=$(find . -name "ghostery-extension-*.zip")
+GHOSTERY_SOURCE_DIR="${GHOSTERY_SOURCE_ZIP%.zip*}"
+CLIQZ_SOURCE_ZIP=$(find . -name "browser-core-*.zip")
+CLIQZ_SOURCE_DIR="${CLIQZ_SOURCE_ZIP%.zip*}"
+
+# Extract
+if [ ! -d $GHOSTERY_SOURCE_DIR ]; then
+	unzip $GHOSTERY_SOURCE_ZIP
+fi
+
+if [ ! -d $CLIQZ_SOURCE_DIR ]; then
+	unzip $CLIQZ_SOURCE_ZIP
+fi
+
+#### BROWSER CORE ####
+cd $CLIQZ_SOURCE_DIR
+
+# Clean any previous builds
+rm -rf build
+
+# Clean all the exiting node_modules for a more reproducible build
+rm -rf node_modules
+
+# Install the exact versions from package-lock.json
+npm ci
+
+# Build a production version of browser-core for Ghostery
+./fern.js build configs/ghostery.js --no-maps --no-debug --environment=production
+
+echo "Browser Core build complete. Please see build/ directory."
+
+cd ..
+
+#### GHOSTERY EXTENSION ####
+cd $GHOSTERY_SOURCE_DIR
+
+VERSION_FILE=manifest.json
+MANIFEST_BACKUP=$(cat $VERSION_FILE)
+RAW_VERSION=$(cat $VERSION_FILE | jq '.version')
+VERSION=${RAW_VERSION//\"} # remove ""
+BUILD_DIR=build
+ZIP_FILE="$BUILD_DIR/ghostery-extension-v$VERSION.zip"
+TMP_FILE=$(mktemp)
+
+# Check for yarn
+if ! type yarn > /dev/null; then
+	abort "Please install yarn: https://yarnpkg.com/lang/en/docs/install/"
+fi
+
+# Check for jq
+if ! type jq > /dev/null; then
+	abort "Please install jq: https://stedolan.github.io/jq/download/"
+fi
+
+# Check for nvm
+source /usr/local/opt/nvm/nvm.sh
+if ! command -v nvm | grep -q 'nvm'; then
+	abort "Please install nvm: https://github.com/nvm-sh/nvm"
+fi
+
+# Clean any previous builds
+rm -rf build
+
+# Clean all the exiting node_modules for a more reproducible build
+rm -rf node_modules
+
+# Set node version
+nvm use
```

Comment:
this line throws `Found '.../ghostery-extension/tools/amo/ghostery-source/ghostery-extension-8.4.1/.nvmrc' with version <lts/carbon>` 
any idea why? can it be due to me changing to version 8.4.1 to use something released ?

Reply:
I think you have to run `nvm install <version>` before `nvm use` can select it.